iphone_connector.py

```diff
@@ -132,15 +132,21 @@ def get_message_df():
     Returns:
         a pandas dataframe representing all text messages
     """
-    messages_df = pd.read_sql_query('''
-    SELECT
-      ROWID as message_id, text, handle_id, country, service, version,
-      DATETIME(date, 'unixepoch', '31 years') AS date,
-      DATETIME(date_read, 'unixepoch', '31 years') AS date_read,
-      DATETIME(date_delivered, 'unixepoch', '31 years') AS date_delivered,
-      is_emote, is_from_me, is_read, is_system_message, is_service_message, is_sent,
-      has_dd_results
-    FROM message''', _message_con)
+    try:
```

Comment:
Yes, it did, and sqlite3.connect() does not catch the encryption error because it only deals with establishing a connection to the DB without any performing any reads, so that's why the exception happens when running a query. I have modified the code as you suggested by running a query at the end of initialize()